src/components/SpotifyPlayer.jsx

```
@@ -0,0 +1,165 @@
+  }
+
+  if (!track) {
+    return (
+      <div style={{ textAlign: "center", color: "var(--ctp-subtext0)" }}>
+        <svg
+          width="64"
+          height="64"
+          viewBox="0 0 24 24"
+          fill="none"
+          style={{ margin: "0 auto 1rem", color: "var(--ctp-green)" }}
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)